.github/workflows/ci.yml

```diff
@@ -305,13 +305,19 @@ jobs:
         run: |
           gcloud auth configure-docker us.gcr.io
 
-      - name: Push to us.gcr.io
+      - name: Push to us.gcr.io and upload deployment assets
         run: |
           set -euxo pipefail
           docker pull -q "$IMG_VERSIONED"
           docker tag "$IMG_VERSIONED" "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}"
           docker push "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}"
 
+          docker run --rm --entrypoint cat "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}" /opt/relay-debug.zip > relay-debug.zip
+          docker run --rm --entrypoint cat "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}" /opt/relay.src.zip > relay.src.zip
+          docker run --rm --entrypoint tar "us.gcr.io/sentryio/relay:${{ github.event.pull_request.head.sha || github.sha }}" -cf - /lib/x86_64-linux-gnu > libs.tar
+          gsutil -m cp ./libs.tar ./relay-debug.zip ./relay.src.zip \
+          "gs://dicd-team-devinfra-cd--getsentry-relay/deployment-assets/${{ github.event.pull_request.head.sha || github.sha }}/"
```

Comment:
Looks CI is failing to upload the artefacts to the bucket. 
If it exists the OPS team will have to update the access permissions for service account which is used here to be able to access this specific bucket.

/cc @oioki

Reply:
Oh yeah, I need to figure out how how to authorize github actions to do this. Cloudbuild was easy because it just ran with a google service account.

Reply:
Oh, nice. Already set up.

```
        uses: google-github-actions/auth@v1
        with:
          workload_identity_provider: projects/345757944225/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider
          service_account: github-actions-gcr@sentryio.iam.gserviceaccount.com```